presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInstaller.java

```diff
@@ -53,19 +62,20 @@ public void runCommand()
     {
         switch (packageType) {
```

Comment:
Any `PackageType` `switch`-block should be replaced by a single method in each installation-manager class.

For now, have the caller (`PackageAPI`) take the `PackageType` as a constructor parameter, and have a single `switch`-block there in which it determines which installation-manager to use. This can be improved upon with Guice later.